src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/LogFormatter.cs

```diff
@@ -28,13 +35,20 @@ public class LogFormatter
         /// </summary>
         private static readonly JsonFormatter formatter = new JsonFormatter(renderMessage: true);
 
+#if NET5_0_OR_GREATER
+        /// <summary>
+        /// Settings to drop null values.
+        /// </summary>
+        private static readonly JsonSerializerOptions settings = new JsonSerializerOptions { WriteIndented = false, DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull, Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping};
+#else
+
         /// <summary>
         /// Settings to drop null values.
         /// </summary>
-        private static readonly JsonSerializerSettings settings = new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore };
+        private static readonly JsonSerializerSettings settings = new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore, Formatting = Newtonsoft.Json.Formatting.None };
```

Comment:
NIT: there's a few places in this PR where the braces have moved to the same line, could you fix the formatting in these cases so they are on a newline?

Reply:
Done